interface/templates/interface/submission_list.html

```diff
@@ -33,7 +33,7 @@
                 {% endif %}
               </td>
               <td>{{ sub.total_score }}/{{ sub.assignment.max_score }}</td>
-              <td>{{ sub.state }}</td>
+              <td>{{ sub.get_state_display }}</td>
```

Comment:
Where is `get_state_display` defined? I can't find it anywhere.

Reply:
Got it [here](https://docs.djangoproject.com/en/3.0/ref/models/instances/#django.db.models.Model.get_FOO_display).